.travis.yml

```diff
@@ -2,8 +2,9 @@ sudo: false
 language: java
 matrix:
   include:
-    - name: Oracle JDK 8
+    - name: Oracle JDK 8 on Trusty
```

Comment:
see jib: https://github.com/GoogleContainerTools/jib/pull/1874

Reply:
I think we should prefer `openjdk8` if it works. In the Jib case, `openjdk8` has a Javadoc-related issue.

Reply:
I think we ran into the same issue with javadoc, that's why we use openjdk, but I don't remember exactly.

Reply:
let me try.